tools/harness-automation/autothreadharness/runner.py

```diff
@@ -98,7 +102,8 @@ def add_result(self, test, passed, error=None):
             'started': self.started,
             'stopped': time.strftime('%Y-%m-%dT%H:%M:%S'),
             'passed': passed,
-            'error': error
+            'error': error,
+            'executions': self.executions
```

Comment:
Use `SimpleTestResult.executions` to reduce confusing?

Reply:
Done